apps/codelab/src/app/codelabs/not-found/not-found.component.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  font-family: 'Helvetica Neue', sans-serif;
+
+  .message {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .left {
```

Comment:
Would be nice to have more semantic names than left and right

Reply:
Renamed